api-spec.md

```diff
@@ -104,16 +104,22 @@ See the [OpenAPI specification document](openapi/STAC.yaml).
 | /search       | [ItemCollection](../item-spec/itemcollection-spec.md) | Retrieves a group of Items matching the 
 provided search predicates, probably containing search metadata from the `search` extension |
 
-The `/` endpoint should function as a complete `Catalog` representation of all the data contained in the API and linked 
-to in some way from root through `Collections` and `Items`.
+The root endpoint (`/`) should function as a complete `Catalog` representation of all the data contained in the API. All `Collections` and `Items` should be linked to from the root through a `data` Link (returning all `Collections`), one or more `child` Links each referencing a single `Collection`, and a `search` Link referencing the `/search` endpoint.
```

Comment:
The `data` rel seems a bit unclear. I think we should just say that data should point to /collections, right? Also, that is a requirement in OAFeat (but not required for a STAC API that only implements search). And then I'm wondering why we should link to all collections in / if there's the pointer to /collections?

Reply:
I thought this was unclear also.  Yes, the example in OAFeat is 
```
    { 
      "href": "http://data.example.org/collections",
      "rel": "data", 
"type": "application/json", 
"title": "Information about the feature collections" }
```

I'm not sure what you mean by "pointer to /collections"?

Reply:
I just reworded it -- I think it's more clear now that most of the endpoints are optional, so the links are also.

Reply:
That reads much better, thanks!